server/src/main/scala/org/http4s/server/middleware/RequestLogger.scala

```diff
@@ -0,0 +1,29 @@
+package org.http4s
+package server
+package middleware
+
+import fs2._
+import org.log4s._
+import cats.implicits._
+import fs2.interop.cats._
+
+/**
+  * Simple Middleware for Logging Requests As They Are Processed
+  */
+object RequestLogger {
```

Comment:
So this is only logging the body.  Should it be called `RequestBodyLogger`?  How would we go about correlating with headers when we log those?